pkg/mediationcontainer/client_websocket_transport.go

```diff
@@ -116,9 +116,6 @@ func (wsTransport *ClientWebSocketTransport) GetConnectionId() string {
 func (wsTransport *ClientWebSocketTransport) CloseTransportPoint() {
 	glog.V(4).Infof("[CloseTransportPoint] closing transport endpoint and listener routine")
 	wsTransport.closeRequested = true
-	// close listener
-	wsTransport.stopListenForMessages()
-	wsTransport.closeAndResetWebSocket()
```

Comment:
Can you explain the reason removing these two lines?  Sorry I'm not immediately clear about why.

Reply:
These two lines are called inside `ListenForMessages()` to close the websocket connection where there are errors reading message from the websocket. This usually happens when `topology-processor` dies for whatever reason (e.g., during an update) causing the connection closed on the server side, in which case, the probe will retry connections with the server.

When a probe registration fails (e.g., due to protocol incompatibility), what we see on the probe side is also a close of connection on the server side:

Reply:
@ading1977 Thanks for the detailed explanation!  That makes sense to me.

I see at least the first line was added by @pallavidn initially with this commit; perhaps she can take a look as well.
https://github.com/turbonomic/turbo-go-sdk/commit/fd1ebc0a3daca871828e943fdf2704b26d6062ce#diff-ab2326914686a36993657f52e15b6884R99

Reply:
Will merge for now as @pallavidn is on PTO.